corehq/apps/cleanup/migrations/0012_add_es_index_to_checkpoint_ids.py

```diff
@@ -60,7 +60,7 @@ class Migration(migrations.Migration):
 
     dependencies = [
         ('cleanup', '0011_merge_couch_sql_pillows'),
-        ('pillowtop', '0003_auto_20170411_1957'),
+        ('pillowtop', '0005_kafkacheckpoint_doc_modification_time'),
```

Comment:
@Rohit25negi why was this changed? I think that this is going to break on prod because this migration has already been applied. locally I saw an error like this:


```
django.db.migrations.exceptions.InconsistentMigrationHistory: Migration cleanup.0012_add_es_index_to_checkpoint_ids is applied before its dependency pillowtop.0005_kafkacheckpoint_doc_modification_time on database 'default'.
```

Reply:
i added the because otherwise was not being included in the migration process.. hmm, i will just see how to correct this.

Reply:
I think 0012_add_es_index_to_checkpoint_ids.py should be rerun as it depends on the pillowtop, and in this Pr i added a migration on pillowtop. so shouldn't all pillowtop's dependents needs to be rerun

Reply:
@calellowitz